test/integration/helpers/Npm.cs

```diff
@@ -0,0 +1,105 @@
+﻿using System;
+using System.ComponentModel;
+using System.Diagnostics;
+using System.Linq;
+
+namespace Appium.Net.Integration.Tests.helpers
+{
+    internal class Npm
+    {
+
+        public static string GetNpmPrefixPath()
+        {
+            string npmPath = GetNpmExecutablePath();
+            string npmPrefixPath = RunCommand(npmPath, "-g root");
+
+            return npmPrefixPath.Trim();
+        }
+
+        private static string RunCommand(string command, string arguments, int timeoutMilliseconds = 30000)
+        {
+            try
+            {
+                using (Process process = new Process
+                {
+                    StartInfo = new ProcessStartInfo
+                    {
+                        FileName = command,
+                        Arguments = arguments,
+                        RedirectStandardOutput = true,
+                        RedirectStandardError = true,
+                        UseShellExecute = false,
+                        CreateNoWindow = true,
+                    }
+                })
+                {
+                    process.Start();
+                    string output = process.StandardOutput.ReadToEnd();
+                    string errorOutput = process.StandardError.ReadToEnd();
+                    _ = process.WaitForExit(timeoutMilliseconds);
+
+                    int exitCode = process.ExitCode;
+                    if (exitCode == 1)
+                    {
+                        throw new NpmUnknownCommandException($"Command: `{arguments}` exited with code {exitCode}. Error: {output}");
+                    }
+                    else if (exitCode != 0)
+                    {
+                        throw new ApplicationException($"Command: `{command}` exited with code {exitCode}. Error: {errorOutput}");
+                    }
+
+                    return output;
+                }
+            }
+            catch (ApplicationException ex)
```

Comment:
this syntax could be simplified: https://code-maze.com/csharp-catch-multiple-exceptions/